eyecite/helpers.py

```diff
@@ -48,20 +49,17 @@ def get_court_by_paren(
     return court_code
 
 
+_this_year = date.today().year
```

Comment:
I'm pretty sure we get cases from next year sometimes from weird courts in late December. 

Maybe:

```suggestion
_next_year = date.today().year + 1
```